packages/x-data-grid-premium/src/hooks/features/clipboard/useGridClipboardImport.ts

```diff
@@ -121,10 +121,16 @@ class CellValueUpdater {
     }
 
     const { apiRef, getRowId } = this.options;
+    const cellParams = apiRef.current.getCellParams(rowId, field);
+    if (!cellParams.isEditable) {
+      return;
+    }
+
     const colDef = apiRef.current.getColumn(field);
-    if (!colDef || !colDef.editable) {
```

Comment:
this should not be removed
both conditions are needed to make the cell editable